src/proxy-animation.js

```diff
@@ -751,7 +751,7 @@ function fractionalStartDelay(details) {
   if (!(details.timeline instanceof ViewTimeline))
     return 0;
 
-  const startTime = details.timeRange.start;
+  const startTime = details?.timeRange?.start || {name: 'cover', offset: CSS.percent(0)};
```

Comment:
Suggest that we consolidate how we pick up the default for the start and end times for CSS and programmatic animations.  For programmatic, we set default values if not specified in the view timeline options.  This code block is essentially introducing a second place where we do "auto" to start and end time mapping.  

There is a case to be made for not filling in defaults in the element.animate path depending on decisions for getComputedTiming.

Reply:
Added a TODO to consolidate after adding support for animation-delay & animation-end-delay.